firefox-ios/Client/Utils/StopWatchTimer.swift

```diff
@@ -0,0 +1,36 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class StopWatchTimer {
```

Comment:
note to self: seems we made this into its own file from `SearchTermGroupsUtility` since we are using it elswehere